### PR TITLE
[Doc] add link, how to set environment variable

### PR DESCRIPTION
**Pull request by sukrosono**
external link how to set environment variable on wikipedia

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ Puppeteer looks for certain environment variables to aid its operations:
 
 - `HTTP_PROXY`, `HTTPS_PROXY`, `NO_PROXY` - defines HTTP proxy settings that are used to download and run Chromium.
 - `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` - do not download bundled Chromium during installation step.
-
+> Don't know how to set Environment Variable? Read [this](https://en.wikipedia.org/wiki/Environment_variable#Assignment)
```
Comment (aslushnikov) [inline docs/api.md]: I'd rather linkify the "environment variables" on line 150

**Comment by sukrosono** [comment]
I signed it as individual contributor

File: docs/api.md (hunk 1 above)
Comment (sukrosono) [reply docs/api.md]: Yeah that was better, thanks. I already made a change

File: docs/api.md [hunk 2]
```diff
@@ -147,11 +147,10 @@ Puppeteer is a Node library which provides a high-level API to control Chromium
 
 ### Environment Variables
 
-Puppeteer looks for certain environment variables to aid its operations:
+Puppeteer looks for certain [environment variables](https://en.wikipedia.org/wiki/Environment_variable#Assignment) to aid its operations:
```
Comment (aslushnikov) [inline docs/api.md]: nit: let's have a link just to wikipedia article

**Review by aslushnikov — APPROVED** [review]

File: docs/api.md (hunk 2 above)
Comment (sukrosono) [reply docs/api.md]: done, BTW what is nit?

**Pull request merged**